test-curves/src/bn384_small_two_adicity/fr.rs

```diff
@@ -102,7 +103,7 @@ impl FpParameters for FrParameters {
     ]);
 
     // (mod - 1) / 2 = 2972938801625915898129258746014768257744324656783561314222519104145798272973804394996417217026588261812051162269696
-    const MODULUS_MINUS_ONE_DIV_TWO: BigInteger = BigInteger([
+    const MODULUS_MINUS_ONE_DIV_TWO: BigInteger = BigInt::<6>([
```

Comment:
I think we can use the `BigInteger::new` method here to avoid specifying the number of limbs. (It's unfortunate that we can't use the type alias as the constructor =()

Reply:
Ditto for the fixes in `curves`

Reply:
One repo needs to move first. It can be algebra.